src/zope/publisher/interfaces/__init__.py

```diff
@@ -24,30 +22,35 @@
 from zope.interface.common.interfaces import IException
 from zope.interface.common.interfaces import ILookupError
 from zope.security.interfaces import IParticipation
-from zope.security.interfaces import Unauthorized
+from zope.security.interfaces import Unauthorized  # noqa: F401 import unused
```

Comment:
Huh.  Now I wonder why the Unauthorized import is here.

Was it once defined in this file?  Was it once used in this file?   Is it kept for backwards compatibility or was it simply overlooked?

(I do not expect you to answer these questions now, but this illustrates why it's useful to keep `BBB` comments around.)

Reply:
It is imported by `zope/publisher/base.py`, so I added a BBB comment.